internal/mailer/mailer.go

```diff
@@ -0,0 +1,6 @@
+package mailer
+
+// Mailer interface
+type Mailer interface {
+	SendSuggestion(email string, html string) error
```

Comment:
Just FYI, on golang interface naming:

```By convention, one-method interfaces are named by the method name plus an -er suffix or similar modification to construct an agent noun: Reader, Writer, Formatter, CloseNotifier etc.

There are a number of such names and it's productive to honor them and the function names they capture. Read, Write, Close, Flush, String and so on have canonical signatures and meanings. To avoid confusion, don't give your method one of those names unless it has the same signature and meaning. Conversely, if your type implements a method with the same meaning as a method on a well-known type, give it the same name and signature; call your string-converter method String not ToString.
```

If you had named your method `Mail()` it would have be just right :P -- don't worry about changing it though.